utilities/transactions/write_prepared_transaction_test.cc

```diff
@@ -3497,6 +3497,31 @@ TEST_P(WritePreparedTransactionTest, WC_WP_WALForwardIncompatibility) {
   CrossCompatibilityTest(WRITE_PREPARED, WRITE_COMMITTED, !empty_wal);
 }
 
+TEST_P(WritePreparedTransactionTest, IngestFile) {
+  // Use large buffer to avoid memtable flush after 1024 insertions
+  options.write_buffer_size = 1024 * 1024;
+  txn_db_options.skip_concurrency_control = true;
+  printf("two write que: %d, unordered write: %d\n", options.two_write_queues, options.unordered_write);
+  ReOpen();
+  ASSERT_OK(db->Put(WriteOptions(), "1000", "v1"));
+  ASSERT_OK(db->Put(WriteOptions(), "1001", "v1"));
+  ASSERT_OK(db->Put(WriteOptions(), "9999", "v1"));
+  ExternalSstFileInfo info;
+  std::string f = test::PerThreadDBPath("tmp_sst_file" + std::to_string(rand()));
```

Comment:
Once this test moved to external_sst_file_test.cc, there you can find helper functions to ingest a file.